documentation/src/main/markdown/releasenotes.md

```diff
@@ -8,6 +8,7 @@
 
 ### Resolved issues
 
+* (IDETECT-3623) [solution_name] will now fail with exit code 3, FAILURE_POLICY_VIOLATION, if Black Duck reports any violated policies during scans.
```

Comment:
One tiny tweak. Please use [blackduck_product_name] in place of "Black Duck". 
Thanks!